types.d.ts

```diff
@@ -23,6 +23,11 @@ declare global {
     var React: typeof import('react')
     var ReactNative: typeof import('react-native')
 
+    var __PYON_LOADER__: {
```

Comment:
Loader-specific things should be implemented under `@revenge-mod/loader/*`. This can probably be removed for now.